packages/db/index.js

```diff
@@ -117,7 +117,7 @@ platformaticDB.configManagerConfig = {
   schema,
   allowToWatch: ['.env'],
   schemaOptions: platformaticService.configManagerConfig.schemaOptions,
-  replaceEnvIgnore: ['$.db.openapi.ignore'],
+  replaceEnvIgnore: ['$.db.openapi.ignoreRoutes'],
```

Comment:
why aren't you keeping both rules? Can you cover this with a test?

Reply:
We don't need it for both rules, because only the `ignoreRoutes` option expects a usage of braces `{}` in the value string.